packaging/pre_build_script_windows.sh

```diff
@@ -6,3 +6,5 @@ python -m pip install tensorrt==10.0.0b6 tensorrt-${CU_VERSION::4}-bindings==10.
 choco install bazelisk -y
 
 cat toolchains/ci_workspaces/WORKSPACE.win.release.tmpl | envsubst > WORKSPACE
+
+echo "RELEASE=1" >> ${GITHUB_ENV}
```

Comment:
How does this related to channel=release in the yml files ?

Reply:
Based on the discussion here: https://github.com/pytorch/TensorRT/pull/2806#discussion_r1591617675, this flag seems to get the package naming correct on the Torch upload index. I do not think it directly relates to the `channel` from the `.yml` files.